### ISSUE TITLE
Pharyngeal Exudate

### ISSUE THREAD

**Issue by cmrn-rhi** [obophenotype/human-phenotype-ontology#7115]
**Preferred term label:**
Pharyngeal exudate

**Synonyms**
pharyngeal fluid
fluid from the pharynx

**Definition (free text, please give PubMed ID)**
Fluid exuded from the pharynx posterior wall.

**Parent term (use hpo.jax.org/app)**
HP:0000600 - Abnormality of the pharynx

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-19

**Your nano-attribution (ORCID)**
0000-0002-9578-0788
